doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+"""
+========================================================================
+Reconstruction of the diffusion signal with the correlation tensor model
+========================================================================
+
+The Correlation Tensor Model (CTI) is an advanced extension of the Diffusion Kurtosis Model (see :ref:sphx_glr_examples_built_reconstruction_reconst_dki.py). [ not sure]
+In addition to the DKI, CTI introduces a more complete representation of the underlying microstructural complexity. Specifically, the CTI differentiates between different sources of kurtosis by distinguishing between isotropic, anisotropic, and microscopic contributions. This refined characterization provides richer insights into tissue microstructure and can reveal features that may be obscured or conflated in simpler models.
+Also, similar to DTI, assumes that the diffusion properties within each Gaussian compartment can be described by a diffusion tensor. However, unlike DTI, CTI allows for the presence of multiple diffusion tensors within a voxel, representing different Gaussian compartments.
+
+In DTI, we’ve diffusion tensor which measure the average diffusion properties within each voxel. in case of DKI we’ve diffusion tensor as well as kurtosis tensor. kurtosis tensor measures the dialation of diffusion properties from the gaussaian behaviour. But in case of CTI we’ve diffusion tensor, kurtosis tensor as well as covariance tensor. The covariance tensor describes the correlation between diffusion measurements at different b-values, offering insights into the relationship between diffusion processes at different scales.
+
+CTI is also very suitable for situations which involve crossing fibers. As unlike DKI, it provides additional information in those cases by providing information about microscopic anisotropy.
+
+[ give maths for formulas ] 
+The CorrelationTensorModel expresses the diffusion-weighted signal in the same way as the DiffusionKurtosisModel. 
+
+However it differentiates from the DiffusionKurtosis Model by calculating the different sources of kurtosis. The isotropic source is calculated as:  
```

Comment:
I think for now we can ignore the maths in this tutorial, the information above is enough for the users. At the moment, if they want to know more they can go to the original papers. Let's keep it simple for this PR and for your GSoC project.